app/controllers/users.js

```diff
@@ -48,3 +52,25 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.adminUser = async (req, res, next) => {
+  try {
+    const { isAdmin } = req.user;
+    if (!isAdmin) throw forbiddenError(forbiddenErrorMessage);
+    const { firstName, lastName, email, password } = req.body;
+    const hashPass = await hashPassword(password);
+    const adminUser = await createAdminUser({
+      firstName,
+      lastName,
+      email,
+      password: hashPass,
+      isAdmin: true
+    });
+    const action = adminUser[1] ? 'created' : 'updated';
+    logger.info(`admin user ${adminUser[0].id} was ${action} succesfully`);
```

Comment:
This would be better in an interactor, as told in #9

Reply:
I already changed it